testing/bsg_cache/lock_test/sweep.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This is nice and thorough, but it takes too long. Can we possibly combine some of these tests or parallelize them?

Reply:
I guess you are trying to collect statistics. This is not a must, but something to consider.

Reply:
@Songchun-Li  I suggest using gnu parallel for this, it's pretty much what it's built for.

https://www.gnu.org/software/parallel/man.html
Here's an example of how we use it in BP
https://github.com/black-parrot/black-parrot/blob/master/ci/single_core_testlist.sh#L41